python_modules/dagster/dagster/_core/definitions/metadata/source_code.py

```diff
@@ -40,6 +41,16 @@ class LocalFileCodeReference(DagsterModel):
     label: Optional[str] = None
 
 
+@experimental
+@whitelist_for_serdes
+class SourceControlCodeReference(DagsterModel):
```

Comment:
Thoughts on calling this `UrlCodeReference`?

Reply:
Seems fine, maybe we should drop the line number and expect the python code to bake that into the URL param in that case.

Reply:
I did that ^

Reply:
> maybe we should drop the line number and expect the python code to bake that into the URL param in that case.

That makes sense